index.html

```diff
@@ -560,17 +560,59 @@ <h4>$schema</h4>
                </p>
             </section>
             <section class="normative">
-               <h4>name</h4>
+               <h4><dfn>title</dfn></h4>
                <p>
-                  It is RECOMMENDED that all JSON Schemas include a <code>name</code> property which provides
-                  a human-readable name that describes the schema.
+                  It is RECOMMENDED that all JSON Schemas include the optional <code>title</code> property as defined in
+                  [[JSON-SCHEMA]].
                </p>
             </section>
             <section class="normative">
-               <h4>description</h4>
+               <h4><dfn>description</dfn></h4>
                <p>
-                  JSON Schemas MAY choose to include an optional <code>description</code> property which provides
-                  a human-readable sentence describing the schema.
+                  It is RECOMMENDED that all JSON Schemas include the optional <code>description</code> property as
+                  defined in [[JSON-SCHEMA]].
+               </p>
+            </section>
+            <section class="normative">
+               <h4>dir</h4>
+               <p>
+                  JSON Schemas MAY choose to include an optional <code>dir</code> property that indicates the
+                  base direction for the values of the [=title=] and [=description=] properties defined in
+                  the JSON Schema. The value of this property MUST be a <a>text-direction</a>. When absent, implementers
+                  MUST assume that the value of this property is "[=text-direction/auto=]".
+               </p>
+              <p>
+                The <dfn>text-directions</dfn> are the following:
+              </p>
+              <dl>
+                <dt>
+                  "<dfn data-dfn-for="text-direction">ltr</dfn>"
+                </dt>
+                <dd>
+                  Left-to-right text.
+                </dd>
+                <dt>
+                  "<dfn data-dfn-for="text-direction">rtl</dfn>"
+                </dt>
+                <dd>
+                  Right-to-left text.
+                </dd>
+                <dt>
+                  "<dfn data-dfn-for="text-direction">auto</dfn>" (default)
+                </dt>
+                <dd>
+                  No explicit directionality.
+                </dd>
+              </dl>
+            </section>
+            <section class="normative">
+               <h4>lang</h4>
+               <p>
+                 JSON Schemas MAY choose to include an optional <code>lang</code> property that indicates the
```

Comment:
Consider making `lang` come before `dir`, as it is the more commonly used property.

I would suggest changing `MAY` to `SHOULD` for both `lang` and `dir`. Having language and direction metadata is a best practice. From a conformance point of view there is no difference, but we might get better adoption with the stronger requirement.